meta-lmp-base/recipes-samples/images/lmp-image-common.inc

```diff
@@ -104,6 +104,6 @@ python () {
 
 EXTRA_USERS_PARAMS = "\
 groupadd ${LMP_USER}; \
-useradd -p '${LMP_PASSWORD}' ${LMP_USER}; \
```

Comment:
Can you check if it gets created when trying to log-in via ssh first (before any serial console login)?

AFAIK it wasn't really able to create automatically when calling via ssh, same issue we have after running recovery.

Reply:
I have tested it only the the serial with lmp and lmp-base. I need to check the ssh too

Reply:
Tested with the lmp-base-console-image and it also works over ssh.